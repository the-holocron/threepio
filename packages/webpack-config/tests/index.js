const config = require('../');

describe('Configuration', () => {
	describe('Webpack', () => {
		it('should render the configuration', () => expect(config).toMatchSnapshot());
	});
});
