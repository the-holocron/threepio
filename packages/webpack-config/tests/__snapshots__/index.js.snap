// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Configuration Webpack should render the configuration 1`] = `
Object {
  "module": Object {
    "rules": Array [
      Object {
        "test": /\\\\\\.js\\$/,
        "use": Object {
          "loader": "babel-loader",
        },
      },
      Object {
        "test": /\\\\\\.md\\$/,
        "use": Object {
          "loader": "raw-loader",
        },
      },
      Object {
        "include": "/Users/chrisopedia/Code/holocron",
        "test": /\\\\\\.scss\\$/,
        "use": Array [
          Object {
            "loader": "style-loader",
          },
          Object {
            "loader": "css-loader",
          },
          Object {
            "loader": "postcss-loader",
          },
          Object {
            "loader": "sass-loader",
            "options": Object {
              "includePaths": Array [
                "node_modules/",
              ],
            },
          },
        ],
      },
    ],
  },
  "node": Object {
    "fs": "empty",
    "net": "empty",
    "tls": "empty",
  },
  "plugins": Array [
    ObsoleteWebpackPlugin {
      "options": Object {
        "name": "obsolete",
        "position": "afterbegin",
        "promptOnNonTargetBrowser": false,
        "promptOnUnknownBrowser": true,
      },
    },
  ],
}
`;
